templates/user/identityForm.tpl

```diff
@@ -51,16 +51,24 @@
 	{/fbvFormSection}
 
 	{if $orcidEnabled}
+
+	<div class="orcid_container">
```

Comment:
I'm not sure if this broke here specifically, but the code in `AuthorizeUserData` that's [supposed to place](https://github.com/pkp/pkp-lib/blob/9dc8a4acc269b3c4f44db5279eb0963273af6fd4/classes/orcid/actions/AuthorizeUserData.php#L130-L142) the ORCID value onto the page and remove the button isn't working anymore.

Reply:
Fixed. The issue was actually [here](https://github.com/pkp/pkp-lib/blob/9dc8a4acc269b3c4f44db5279eb0963273af6fd4/classes/orcid/actions/AuthorizeUserData.php#L154). It was still attempting to refresh the "public"(index 3) tab in the user profile. I think it's a regression from this [PR](https://github.com/pkp/pkp-lib/pull/10325) that moved the ORCID stuff from the "public" tab to "identify"